actions/telemetry_actions.jsx

```diff
@@ -92,22 +99,70 @@ export function measure(name1, name2) {
     return [lastDuration, measurementName];
 }
 
-export function trackLoadTime() {
+function trackPageLoadTimeEvent() {
     if (!isSupported([performance.timing.loadEventEnd, performance.timing.navigationStart])) {
         return;
     }
 
+    const {loadEventEnd, navigationStart} = window.performance.timing;
+    const pageLoadTime = loadEventEnd - navigationStart;
+    trackEvent('performance', 'page_load', {duration: pageLoadTime});
+}
+
+/**
+ * Measures the time and number of requests on first page load.
+ */
+export function measurePageLoadTelemetry() {
     // Must be wrapped in setTimeout because loadEventEnd property is 0
     // until onload is complete, also time added because analytics
     // code isn't loaded until a subsequent window event has fired.
     const tenSeconds = 10000;
     setTimeout(() => {
-        const {loadEventEnd, navigationStart} = window.performance.timing;
-        const pageLoadTime = loadEventEnd - navigationStart;
-        trackEvent('performance', 'page_load', {duration: pageLoadTime});
+        trackPageLoadTimeEvent();
+        trackNumOfRequestEvent('page_load');
```

Comment:
Perhaps instead of having two separate events for this, we could add these into the events for page_load. channel_switch, and team_switch? So instead of sending just `duration`, we also add `totalRequests` and `maxResourceSize`

Reply:
not handling in this pr